helm/hpcc/templates/sasha.yaml

```diff
@@ -28,6 +28,10 @@ spec:
     spec:
       {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" $serviceName "type" "sasha") | indent 6 }}
       serviceAccountName: "hpcc-default"
+      {{- if $sasha.storage }}
+      initContainers:
+        {{- include "hpcc.checkSashaMount" (dict "root" $ "me"  $sasha) | indent 6 }}
```

Comment:
super trivial: extra space before $sasha

Reply:
indent 6 works, but to match layout, should be indent 8

Reply:
Fixed

Reply:
I tested the indent still should be "6"

Reply:
it can be 6 or 8 in this context, we are not consistent whether lists appear intended to parent or level, e.g.:
```
        volumeMounts:
          - name: "dali"    <<<--- indented
            mountPath: "/var/lib/HPCCSystems/dalistorage"
      containers:
      - name: "mydali" <<<--- not indented
```

we should be consistent.
We normally do not indent - so 6 would be correct in this context.

But also, you have 8 spaces before the {{ , whic {{- will throw away), so that's inconsistent as well.